_pages/docs/timeline.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+permalink: /timeline
+last_modified_at: 2024-02-19
+toc: true
+sidebar:
+  nav: "docs"
+---
+
+# Timeline
+
+## What is the timeline?
+
+The timeline is a tool designed to simplify the visualization of a process as a whole, enabling users with less familiarity in BPMN to understand the progress of the process in a linear manner. This allows tracking of ongoing tasks, completed tasks, and those that are still pending.
+
+## How to use the timeline?
+
+### 1. Accessing the timeline of a process
+
+To set up the timeline, you first need to decide which process it will be used for and then access the listing and configuration through the Process Definition screen, located under the 'Timelines' button.
+
+#### Button Timelines
+![Button timelines](assets/images/timeline/button-timelines.png)
+
+### 2. Timelines
+
+When you click the button, you'll be directed to a page listing the information of the timelines created for that process. On this page, you can <b>view</b>, <b>edit</b>, and <b>delete</b> a specific timeline. If you wish to configure a new timeline, simply click on the 'Create a new timeline' button.
+
+
+#### Timelines page
+![Timelines page](assets/images/timeline/timelines-page.png)
+
+#### Timeline view
+![Timelines view](assets/images/timeline/timeline-view.png)
+
+#### Timeline edit
+![Timelines edit](assets/images/timeline/timeline-edit.png)
+
+#### Timeline delete
+![Timelines delete](assets/images/timeline/timeline-remove.png)
+
+### 3. Creating a new timeline
```

Comment:
You should include in this section a subsection explaining that you can include, remove, or change the order of the steps that compose the timeline.